data/build.gradle

```diff
@@ -2,6 +2,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply plugin: 'jacoco'
+
+jacoco {
+  toolVersion = "0.8.2"
+  version "0.7.1.201405082137"
+}
```

Comment:
We should define this in one location to reduce potential error in the future if we ever change the version. It can be defined in one location similar to Kotlin version: https://github.com/oppia/oppia-android/blob/develop/build.gradle#L4.

Ditto elsewhere.

Reply:
Done